content/departments/product-engineering/engineering/cloud/delivery/on-call.md

```diff
@@ -0,0 +1,126 @@
+# Delivery Team On-Call Rotation
+
+The Delivery Team maintains an on-call rotation where an engineer is dedicated to providing support to our
+internal teams and managed instances for a term of 1 week.
+
+The on-call schedule is captured in OpsGenie. The on-call rotation consists of an Engineer On-Call (EOC), a Secondary Engineer On-Call (SEOC), and a technical manager (TM).
+
+On-Call duties transfer automatically Mondays at 10am Central Time according to the schedule within OpsGenie.
+
+## Response Times and SLAs
+
+The Delivery team aims to provide timely responses that are proportional to the capacity of the team.
+The Engineer On-Call (EOC) is responsible for a 24 hour x 7 period shift, guarenteeing responses during their
+working hours (generally 9am - 5pm in their timezone).
+
+The EOC is expected to provide an initial response within 45 minutes in response to an Incident where they are activated with an OpsGenie page. If the EOC is unavailable within the first 45 minutes of an OpsGenie page, the Secondary Engineer On-Call will be activated, where the expectation is to provide an initial response within an additional 45 minutes.
```

Comment:
The opsgenie rotation is actually set up to escalate to a secondary after 10 minutes, then admins 5 minutes later, then everyone on delivery 5 minutes after that.

i.e. We currently "guarantee" responses within 20 minutes of being paged.

Since most of our alerting isn't tied to opsgenie, this feels reasonable - we'd only be paged in cases of actual emergencies, and a quick response / escalation process seems appropriate.

Reply:
Gotcha! This has been updated.